taxonomies/labels.txt

```diff
@@ -18479,8 +18557,10 @@ hu:Új
 it:Nuovo
 nl:Nieuw
 pl:Nowość, Nowy, Nowe
+pt:Novo
 ru:Новинка
 th:สินค้าใหม่
+tr:Yeni
 
 en:New recipe
```

Comment:
Why new recipe and nutriscore?

Reply:
sometimes product change, and new recipe is displayed on the packaging